packages/govuk-frontend/src/govuk/components/table/table.yaml

```
@@ -35,7 +35,7 @@ params:
   - name: head
     type: array
     required: false
-    description: Array of table head cells.
+    description: The head cells within the table component.
```

This macro isn't great – reality is that [table header cells can appear anywhere in a table](https://www.w3.org/WAI/tutorials/tables/) but our `head` option just adds a row at the very top of the table and makes all the cells within it a header, so it's really only useful for very simple tables.

I don't think we can solve this completely with content, but we could follow the pattern we've used elsewhere to try and explain what it does?

```suggestion
    description: Can be used to add a row of table header cells (`<th>`) at the top of the table component.
```

We might also want to tweak the description used for `firstCellIsHeader` to talk about 'table headers' instead:

> If set to true, the first cell in each row will be a table header (`<th>`).